nixos/modules/services/networking/centrifugo.nix

```diff
@@ -65,6 +65,13 @@ in
   };
 
   config = lib.mkIf cfg.enable {
+    assertions = [
+      {
+        assertion = !(cfg.settings ? name) && !(cfg.settings ? port);
```

Comment:
Perhaps add an additional `lib.versionAtLeast cfg.package.version "6" -> …` condition since it’s technically possible to use this module with v5?